etc/kayobe/environments/ci-aio/inventory/group_vars/cis-hardening/cis

```diff
@@ -14,3 +14,10 @@ rhel9cis_rule_5_4_3_2: false
 ubtu22cis_rule_5_4_3_2: false
 
 ##############################################################################
+# Ubuntu Noble CIS Hardening Configuration
+
+# Disable shell timeout for inactivity which can be disruptive to
+# development work.
+ubtu24cis_rule_5_4_3_2: false
```

Comment:
The rules often change, have you checked this is the right rule?

Reply:
I've seen you've mentioned on the CIS file that you haven't tested the variables so that answers this question, probably best to just add a TODO for now

Reply:
Do you mean removing the rules then just add TODO to add them later or add TODO to testing the rules?

Reply:
Just add a todo commentto test the rules